tests/e2e/config/default.json

```diff
@@ -25,6 +25,11 @@
       "externalUrl": "https://wordpress.org/plugins/woocommerce"
     }
   },
+  "coupon": {
```

Comment:
Can we make this an indexed array for future use in the cart/checkout tests? Something like `percentage`.

Reply:
I debated on doing this as well, and I agree that would be idea so we can add more coupon types. Added :)